.vortex/tests/phpunit/Functional/ToolingBootstrapTest.php

```
@@ -130,7 +162,7 @@ public function testComposerInstallAfterBootstrap(): void {
     $this->logStepFinish();
   }
 
-  protected function prepareProject(bool $minimal_composer_json = FALSE): string {
+  protected function prepareProject(bool $minimal_composer_json = FALSE, ?string $tooling_constraint = NULL): string {
```

PHPMD (2.15.0)

[error] The method prepareProject has a boolean flag argument $minimal_composer_json, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)